docs/sources/setup/install/helm/deployment-guides/aws.md

```diff
@@ -0,0 +1,763 @@
+                    "s3:GetObject",
+                    "s3:DeleteObject",
+                    "s3:ListBucket"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::<ruler bucket name>",
+                    "arn:aws:s3:::<ruler bucket name>/*"
+                ]
+            }
+        ]
+    }
+    ```
+    **Make sure to replace the placeholders with your AWS account ID and the bucket name.**
+
+1. Add the policy to the bucket:
+
+    ```bash
+    aws s3api put-bucket-policy --bucket <Your S3 bucket eg. `loki-aws-dev-ruler`> --policy file://bucket-policy-ruler.json
+    ```
+
+## Deploying the Helm chart
+
+Before we can deploy the Loki Helm chart, we need to add the Grafana chart repository to Helm. This repository contains the Loki Helm chart.
+
+1. Add the Grafana chart repository to Helm:
+
+    ```bash
+    helm repo add grafana https://grafana.github.io/helm-charts
+    ```
+1. Update the chart repository:
+
+    ```bash
+    helm repo update
+    ```
+1. Create a new namespace for Loki:
+
+    ```bash
+    kubectl create namespace loki
+    ```
+### Loki Basic Authentication
+
+Loki by default does not come with any authentication. Since we will be deploying Loki to AWS and exposing the gateway to the internet, we recommend adding atleast basic authentication. In this guide we will give Loki a username and password:
+
+1. To start we will need create a `.htpasswd` file with the username and password. You can use the `htpasswd` command to create the file:
+   
+   {{< admonition type="tip" >}}
+    If you don't have the `htpasswd` command installed, you can install it using `brew` or `apt-get` or `yum` depending on your OS.
+   {{< /admonition >}}
+
+    ```bash
+    htpasswd -c .htpasswd <username>
+    ```
+    This will create a file called `auth` with the username `loki`. You will be prompted to enter a password.
+
+ 1. Create a Kubernetes secret with the `.htpasswd` file:
+
+    ```bash
+    kubectl create secret generic loki-basic-auth --from-file=.htpasswd -n loki
+    ```
+
+    This will create a secret called `loki-basic-auth` in the `loki` namespace. We will reference this secret in the Loki Helm chart configuration.
+
+    
+
+### Loki Helm chart configuration
+
+Create a `values.yaml` file choosing the configuration options that best suit your requirements. Below there are two examples of `values.yaml` files for the Loki Helm chart. The first template is Loki in [microservices](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/deployment-modes/#microservices-mode) mode the second is Loki in [monolithic](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/deployment-modes/#monolithic-mode) mode with a replication factor of 3. The rest of this guide will focus on deploying Loki in microservices mode. However, there is no difference in the deployment process between the two modes apart from the components you are deploying.
+
+{{< collapse title="Distributed" >}}
```

Comment:
Since you use the term "microservices" throughout the rest of the doc, I think it makes sense to use it here too.
```suggestion
{{< collapse title="Microservices" >}}
```